taxcalc/functions.py

```diff
@@ -289,14 +292,17 @@ def ItemDed(_posagi, e17500, e18400, e18500, e18800, e18900,
 
     # Gross Itemized Deductions
 
-    c21060 = (e20900 + c17000 + (1 - ID_StateLocalTax_HC) * c18300 + c19200 +
-              c19700 + c20500 + c20800 + e21000 + e21010)
+    c21060 = (e20900 + (1 - ID_Medical_HC) * c17000 +
+              (1 - ID_StateLocalTax_HC) * c18300 +
+              (1 - ID_Mortgage_HC) * c19200 +
+              (1 - ID_Casualty_HC) * c19700 +
+              (1 - ID_Casualty_HC) * c20500 +
+              (1 - ID_Miscellaneous_HC) * c20800 +
```

Comment:
It's a good idea to treat `c20800` and `c17000` indifferently, like we did for `_statax`, in AMTI. A quick question come across that should we use `c60200` in stead of `min(c17000, 0.025 * _posagi) = min(c17000, 0.025 * max(0., c00100)) = min((1 - ID_Medical_HC) * c17000, 0.025 * max(0., c00100))` in the calculation of `c62100`?